dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -147,6 +148,27 @@ RUN dpkg -i \
 debs/{{ deb }}{{' '}}
 {%- endfor %}
 
+# Install protobuf 3.21.12
+RUN mkdir -p /tmp/protobuf \
+    cd /tmp/protobuf \
+    && wget https://sonicstorage.blob.core.windows.net/public/ubuntu/20.04/libprotobuf-dev_3.21.12-3_amd64.deb \
```

Comment:
>20.04 [](http://example.com/codeflow?start=69&length=5)

It is also used in FROM in the beginning of the file. Will dpkg fail if we upgrade the FROM line to new version?

Reply:
These packets are compiled in ubuntu20.04. Actually, a better method is to install these packets from bullseye slave by sonic_mgmt_debs variable, but due to bullseye and ubuntu dependencies issues, like python 3.8 and python 3.9, we have to install a pre-compiled version.
If we upgrade this docker to a new version, like ubuntu22.04, we can follow the typical method to build this docker.

Reply:
Please consider 20.04 as a named constant.